.expeditor/scripts/shared.ps1

```diff
@@ -1,3 +1,13 @@
+$env:PathSeparator = if ($IsWindows) {
```

Comment:
We want to make sure this works on "Windows Powershell" as well. So this should be `if ($IsWindows -Or !$IsCoreCLR)`

Reply:
Because these scripts are only used for testing, do we gain much by supporting "Windows Powershell"? This specific instance is easy to fix, but we lose out on other nice to haves. For instance, `Join-Path`s ability to take multiple arguments, the `Test-Connection` discrepancy below, and using the `&` operator. I am sure there are others. We could certainly do without these, but if we could standardize on powershell core it would make writing these scripts cross platform easier.

Reply:
Both [build.ps1](https://github.com/habitat-sh/habitat/blob/master/build.ps1) and [e2e_local.ps1](https://github.com/habitat-sh/habitat/blob/master/e2e_local.ps1) which source shared.ps1 are part of a local windows developer workflow where one will just use the native ps console. I use them all the time.

Reply:
build.ps1 is basically a powershell make file

Reply:
I see. Maybe we can have a different "shared" script for the e2e tests or probably it should just be part of `run_e2e_test.ps1`? Would we be ok running the e2e test solely on powershell core? `shared.ps1` would stay completely "Windows Powershell" compatible, but developers of the e2e tests would not need to be concerned with the nuances.

Reply:
yeah that sounds like a good idea. We may need to have `run_e2e_test.ps1` load the powershell core package and launch pwsh (powershell core) because right now all of our tests call `run_e2e_test.ps1` from `powershell` (windows powershell).